app/forms/admission_webhook_form.rb

```diff
@@ -0,0 +1,42 @@
+# AdmissionWebhookForm represents a form for AdmissionWebhookController
+class AdmissionWebhookForm
+  include ActiveModel::Model
+
+  attr_accessor :enabled, :certificate, :key, :current_cert, :current_key
+
+  validates :key, presence: true, if: :current_key_blank?
+  validates :certificate, presence: true, if: :current_certificate_blank?
+
+  validate :cant_be_empty_file
```

Comment:
should we at least validate the filetype as well?

```ruby
def must_be_key
  `file #{path}`.include? "private key"
end

def must_be_cert
  `file #{path}`.include? "certificate"
end
```

otherwise i'm afraid we'll have an uncaught exception

Reply:
as discussed on irc we can address this in a follow up, as we have other cert handling as well, and it should be done for all parts where we upload certs